src/cli_plugin/install/download.js

```diff
@@ -8,8 +9,10 @@ export function _downloadSingle(settings, logger, sourceUrl) {
 
   if (/^file/.test(urlInfo.protocol)) {
     downloadPromise = downloadLocalFile(logger, decodeURI(urlInfo.path), settings.tempArchiveFile);
-  } else {
+  } else if (/^https?/.test(urlInfo.protocol)) {
```

Comment:
Note that this is checking against the value of `urlInfo.protocol`, which is the result of parsing the raw sourceUrl (i.e. plugin name) that is passed in. `urlInfo.protocol` looks like `http:`, `https:`, `file:`, or `null` (when a bare string w/o protocol is used).